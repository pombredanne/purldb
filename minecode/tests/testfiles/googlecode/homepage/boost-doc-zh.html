

<!DOCTYPE html>
<html>
<head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" >
 <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1" >
 
 <meta name="ROBOTS" content="NOARCHIVE">
 
 <link rel="icon" type="image/vnd.microsoft.icon" href="http://www.gstatic.com/codesite/ph/images/phosting.ico">
 
 
 <script type="text/javascript">
 
 (function(){(function(){function c(a){this.t={};this.tick=function(a,b,c){b=void 0!=c?c:(new Date).getTime();this.t[a]=b;if(void 0==c)try{window.console.timeStamp("CSI/"+a)}catch(d){}};this.tick("start",null,a)}var a;window.performance&&(a=window.performance.timing);var b=a?new c(a.responseStart):new c;window.jstiming={Timer:c,load:b};a&&(b=a.navigationStart,a=a.responseStart,0<b&&a>=b&&(window.jstiming.srt=a-b));try{a=null,window.chrome&&window.chrome.csi&&(a=Math.floor(window.chrome.csi().pageT)),null==a&&
window.gtbExternal&&(a=window.gtbExternal.pageT()),null==a&&window.external&&(a=window.external.pageT),a&&(window.jstiming.pt=a)}catch(d){}})();})();

 
 
 
 
 var codesite_token = null;
 
 
 var CS_env = {"assetHostPath": "http://www.gstatic.com/codesite/ph", "projectName": "boost-doc-zh", "loggedInUserEmail": null, "projectHomeUrl": "/p/boost-doc-zh", "profileUrl": null, "domainName": null, "assetVersionPath": "http://www.gstatic.com/codesite/ph/11512667522589661435", "relativeBaseUrl": "", "token": null};
 var _gaq = _gaq || [];
 _gaq.push(
 ['siteTracker._setAccount', 'UA-18071-1'],
 ['siteTracker._trackPageview']);
 
 (function() {
 var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
 ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
 (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(ga);
 })();
 
 </script>
 
 
 <title>
 boost-doc-zh -
 
 
 boost 文档汉化 - Google Project Hosting
 </title>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/core.css">
 
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/ph_detail.css" >
 
 
 
 
<!--[if IE]>
 <link type="text/css" rel="stylesheet" href="http://www.gstatic.com/codesite/ph/11512667522589661435/css/d_ie.css" >
<![endif]-->
 <style type="text/css">
 .menuIcon.off { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -42px }
 .menuIcon.on { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 -28px }
 .menuIcon.down { background: no-repeat url(http://www.gstatic.com/codesite/ph/images/dropdown_sprite.gif) 0 0; }
 
 
 .psicon-container {
 min-width:24px;
 }

 
 </style>
</head>
<body class="t1">
<script type="text/javascript">
 window.___gcfg = {lang: 'en'};
 (function() 
 {var po = document.createElement("script");
 po.type = "text/javascript"; po.async = true;po.src = "https://apis.google.com/js/plusone.js";
 var s = document.getElementsByTagName("script")[0];
 s.parentNode.insertBefore(po, s);
 })();
</script>
<div class="headbg">

 <div id="gaia">
 

 <span>
 
 
 <a href="#" id="projects-dropdown" onclick="return false;"><u>My favorites</u> <small>&#9660;</small></a>
 | <a href="https://www.google.com/accounts/ServiceLogin?service=code&amp;ltmpl=phosting&amp;continue=http%3A%2F%2Fcode.google.com%2Fp%2Fboost-doc-zh%2F&amp;followup=http%3A%2F%2Fcode.google.com%2Fp%2Fboost-doc-zh%2F" onclick="_CS_click('/gb/ph/signin');"><u>Sign in</u></a>
 
 </span>

 </div>

 <div class="gbh" style="left: 0pt;"></div>
 <div class="gbh" style="right: 0pt;"></div>
 
 
 <div style="height: 1px"></div>
<!--[if lte IE 7]>
<div style="text-align:center;">
Your version of Internet Explorer is not supported. Try a browser that
contributes to open source, such as <a href="http://www.firefox.com">Firefox</a>,
<a href="http://www.google.com/chrome">Google Chrome</a>, or
<a href="http://code.google.com/chrome/chromeframe/">Google Chrome Frame</a>.
</div>
<![endif]-->



 <table style="padding:0px; margin: 0px 0px 10px 0px; width:100%" cellpadding="0" cellspacing="0"
 itemscope itemtype="http://schema.org/CreativeWork">
 <tr style="height: 58px;">
 
 
 
 <td id="plogo">
 <link itemprop="url" href="/p/boost-doc-zh">
 <a href="/p/boost-doc-zh/">
 
 <img src="http://www.gstatic.com/codesite/ph/images/defaultlogo.png" alt="Logo" itemprop="image">
 
 </a>
 </td>
 
 <td style="padding-left: 0.5em">
 
 <div id="pname">
 <a href="/p/boost-doc-zh/"><span itemprop="name">boost-doc-zh</span></a>
 </div>
 
 <div id="psum">
 <a id="project_summary_link"
 href="/p/boost-doc-zh/"><span itemprop="description">boost 文档汉化</span></a>
 
 </div>
 
 
 </td>
 <td style="white-space:nowrap;text-align:right; vertical-align:bottom;">
 
 <form action="/hosting/search">
 <input size="30" name="q" value="" type="text">
 
 <input type="submit" name="projectsearch" value="Search projects" >
 </form>
 
 </tr>
 </table>

</div>

 
<div id="mt" class="gtb"> 
 <a href="/p/boost-doc-zh/" class="tab active">Project&nbsp;Home</a>
 
 
 
 
 <a href="/p/boost-doc-zh/downloads/list" class="tab ">Downloads</a>
 
 
 
 
 
 <a href="/p/boost-doc-zh/w/list" class="tab ">Wiki</a>
 
 
 
 
 
 <a href="/p/boost-doc-zh/issues/list"
 class="tab ">Issues</a>
 
 
 
 
 
 <a href="/p/boost-doc-zh/source/checkout"
 class="tab ">Source</a>
 
 
 
 
 
 
 
 
 <div class=gtbc></div>
</div>
<table cellspacing="0" cellpadding="0" width="100%" align="center" border="0" class="st">
 <tr>
 
 
 
 
 
 
 
 <td class="subt">
 <div class="st1">
 <div class="isf">
 <span class="inst1">
 <a href="/p/boost-doc-zh/">Summary</a>
 </span>
 
 
 &nbsp;
 <span class="inst3">
 <a href="/p/boost-doc-zh/people/list">People</a>
 </span>
 
 
 </div>
</div>

 </td>
 
 
 <td align="right" valign="top" class="bevel-right"></td>
 </tr>
</table>


<script type="text/javascript">
 var cancelBubble = false;
 function _go(url) { document.location = url; }
</script>
<div id="maincol"
 
>

 





<table width="100%">
 <tr class="pscontent">
 <td class="pscolumnl">
 
 <div class="phead">Project Information</div>
 
 
 
 <ul class="pslist">
 
 <li class="psmeta">
 <a href="/p/boost-doc-zh/feeds">Project feeds</a>
 </li>
 <li class="psgap"/>
 
 
 
 <li class="psmeta"><b>Code license</b></li>
 <li class="psmeta">
 <a href="http://code.google.com/p/support/wiki/FAQ#Which_licenses_can_be_used?" rel="nofollow">Other Open Source</a>
 <br>See source for details
 </li>
 <li class="psgap"/>
 
 
 
 
 
 
 <li class="psmeta">
 <span id="project_labels">
 <b>Labels</b><br/>
 
 <a class="label"
 href="/hosting/search?q=label:boost">boost</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:documentation">documentation</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:translation">translation</a>, 
 
 <a class="label"
 href="/hosting/search?q=label:Chinese">Chinese</a>
 
 </span>
 </li>
 <li class="psgap"/>
 
 
 </ul>
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_people-y16 goog-inline-block vt"></div>
 </div>
 <span><b>Members</b></span>
 </div>
 <ul class="pslist">
 
 
 
 

 <a class="userlink"
 href="/u/105205975368400385253/">xiaqq...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/118268467387985900681/">ala...@gmail.com</a
>, 
 
 

 <a class="userlink"
 href="/u/103520369014801395303/">jinq0...@163.com</a
>
 
 
 
 <li class="psmeta">
 <a href="people/list">29 committers</a>
 </li>
 
 
 <li class="psgap"/>
 
 
 </ul>
 
 
 <div class="phead pfeatured">Featured</div>
 
 <div class="psicon">
 <div class="psicon-container goog-inline-block">
 <div style="float:right" class="SPRITE_filedownload-g16 goog-inline-block vt"></div>
 </div>
 <span><b>Downloads</b></span>
 </div>
 <ul class="pslist">
 
 <li class="psmeta nowrap">
 <a title="Download" href="//boost-doc-zh.googlecode.com/files/Beyond_STL_cn.chm" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//boost-doc-zh.googlecode.com/files/Beyond_STL_cn.chm', 'Featured']);">
 </a>
 <a href="downloads/detail?name=Beyond_STL_cn.chm"
 title="《Beyond the C++ Standard Library》中译本">Beyond_STL_cn.chm</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//boost-doc-zh.googlecode.com/files/boost_1_41_0_doc_20091225.7z" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//boost-doc-zh.googlecode.com/files/boost_1_41_0_doc_20091225.7z', 'Featured']);">
 </a>
 <a href="downloads/detail?name=boost_1_41_0_doc_20091225.7z"
 title="1.41.0版本的中文文档20091225，已完成翻译91个库，7z格式的html包">boost_1_41_0_doc_20091225.7z</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//boost-doc-zh.googlecode.com/files/boost_1_41_0_doc_en.chm" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//boost-doc-zh.googlecode.com/files/boost_1_41_0_doc_en.chm', 'Featured']);">
 </a>
 <a href="downloads/detail?name=boost_1_41_0_doc_en.chm"
 title="Boost 1.41.0 英文文档">boost_1_41_0_doc_en.chm</a>
 </li>
 
 <li class="psmeta nowrap">
 <a title="Download" href="//boost-doc-zh.googlecode.com/files/boost_1_43_0_doc_en.chm" onclick="_gaq.push(['siteTracker._trackEvent', 'Download - Downloaded', '//boost-doc-zh.googlecode.com/files/boost_1_43_0_doc_en.chm', 'Featured']);">
 </a>
 <a href="downloads/detail?name=boost_1_43_0_doc_en.chm"
 title="Boost 1.43.0 英文文档">boost_1_43_0_doc_en.chm</a>
 </li>
 
 <li class="psmeta"><a href="downloads/list">Show all &raquo</a></li>
 <li class="psgap"/>
 </ul>
 
 
 
 
 
 <div class="phead">Links</div>
 <ul class="pslist" style="white-space:nowrap">
 
 
 <li class="psmeta"><b>External links</b></li>
 
 <li class="psmeta">
 <a href="http://www.boost.org/LICENSE_1_0.txt" rel="nofollow">Boost Software License, Version 1.0</a>
 </li>
 
 <li class="psgap"/>
 
 
 <li class="psmeta"><b>Groups</b></li>
 
 <li class="psmeta">
 <a href="http://groups.google.com/group/boost_doc_translation" rel="nofollow">boost_doc_translation</a>
 </li>
 
 <li class="psgap"/>
 
 </ul>
 
 </td>
 <td id="wikicontent" class="psdescription">
 <p>汉化 boost 文档，致力 boost 推广。 </p><p>boost 文档以 Boost Software License, Version 1.0 发布。 </p><p>注意：如果您在 Linux 环境下阅读本站的 chm，推荐使用 chmsee (其它工具可能会有乱码问题)。 </p><p>如果你对本项目有兴趣，欢迎加入，相关说明请查阅项目论坛：<a href="https://groups.google.com/group/boost_doc_translation?hl=zh-CN" rel="nofollow">https://groups.google.com/group/boost_doc_translation?hl=zh-CN</a> </p>
 </td>
 </tr>
</table>
<script src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/prettify/prettify_core_compiled.js"></script>
<script type="text/javascript">
 prettyPrint();
</script>

 
 
 
 <script type="text/javascript" src="http://www.gstatic.com/codesite/ph/11512667522589661435/js/ph_core.js"></script>
 
 
 
 
</div> 

<div id="footer" dir="ltr">
 <div class="text">
 <a href="/projecthosting/terms.html">Terms</a> -
 <a href="http://www.google.com/privacy.html">Privacy</a> -
 <a href="/p/support/">Project Hosting Help</a>
 </div>
</div>
 <div class="hostedBy" style="margin-top: -20px;">
 <span style="vertical-align: top;">Powered by <a href="http://code.google.com/projecthosting/">Google Project Hosting</a></span>
 </div>

 
 


 
 
 <script type="text/javascript">_CS_reportToCsi();</script>
 
 </body>
</html>

