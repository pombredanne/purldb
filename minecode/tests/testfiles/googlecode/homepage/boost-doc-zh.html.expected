{'content_license': None,
 'content_license_url': None,
 'description': u'boost \u6587\u6863\u6c49\u5316',
 'downloads_url': u'http://code.google.com/p/boost-doc-zh/downloads/list',
 'external_links': {},
 'feeds_url': u'http://code.google.com/p/boost-doc-zh/feeds',
 'home_url': u'http://code.google.com/p/boost-doc-zh/',
 'id': u'boost-doc-zh',
 'issues_url': u'http://code.google.com/p/boost-doc-zh/issues/list',
 'keywords': [],
 'labels': [u'Chinese', u'boost', u'documentation', u'translation'],
 'license': u'Other Open Source',
 'license_url': u'http://code.google.com/p/support/wiki/FAQ#Which_licenses_can_be_used?',
 'name': u'boost-doc-zh',
 'peoples_url': u'http://code.google.com/p/boost-doc-zh/people/list',
 'vcs_url': u'http://code.google.com/p/boost-doc-zh/source/checkout',
 'wiki_url': u'http://code.google.com/p/boost-doc-zh/w/list'}